presidio-analyzer/presidio_analyzer/recognizer_registry/recognizer_registry.py

```diff
@@ -206,3 +209,52 @@ def remove_recognizer(self, recognizer_name: str) -> None:
             recognizer_name,
         )
         self.recognizers = new_recognizers
+
+    def add_pattern_recognizer_from_dict(self, recognizer_dict: Dict):
+        """
+        Load a pattern recognizer from a Dict into the recognizer registry.
+
+        :param recognizer_dict: Dict holding a serialization of an PatternRecognizer
+
+        :example:
+        >>> registry = RecognizerRegistry()
+        >>> recognizer = { "name": "Titles Recognizer", "supported_language": "de","supported_entity": "TITLE", "deny_list": ["Mr.","Mrs."]} # noqa: E501
+        >>> registry.add_pattern_recognizer_from_dict(recognizer)
+        """
+
+        recognizer = PatternRecognizer.from_dict(recognizer_dict)
+        self.add_recognizer(recognizer)
+
+    def add_recognizers_from_yaml(self, yml_path: Union[str, Path]):
+        r"""
```

Comment:
NIT: maybe put a link to the readme and doc instead of copy pasting the entire yaml file?

Reply:
It's useful when you work in an IDE and look up the docstring of a method. Usually links don't work in this scenario.